frontend/src/components/SaveSearchButton.jsx

```diff
@@ -1,28 +1,20 @@
-import React, { useState } from "react";
-import heartIcon from '../images/heart.svg';
-import heartRedIcon from '../images/redheart.svg';
-import "../style/Heart.css";
-
-export default function SaveSearchButton({ onSave, onUnsave }) {
-  const [isHeartClicked, setIsHeartClicked] = useState(false);
-
-  const toggleHeart = () => {
-    if (isHeartClicked) {
-      onUnsave();
-    } else {
-      onSave();
-    }
-    setIsHeartClicked((prevIsHeartClicked) => !prevIsHeartClicked);
-  };
+import "../style/Heart.css"
```

Comment:
There's a class in the Heart.css file called heart icon that has a transition on fill that might be nice to keep, although we can ditch the height and width from that class.

Reply:
done